src/validations/test/ssp.xspec

```diff
@@ -6668,11 +6668,11 @@
                             xmlns="http://csrc.nist.gov/ns/oscal/1.0">
                             <prop 
                                 name="ipv6-address" 
-                                value="1234:5678:9012:3456"/>
+                                value="2001:0db8:85a3:0000:0000:8a2e:0370:7334"/>
                         </inventory-item>
                     </x:context>
                     <x:expect-assert
-                        id="inventory-item-has-empty-ipv6"
+                        id="ipv6-has-non-placeholder"
```

Comment:
Should this be `id="ipv6-has-placeholder"`? (since the assertion is expected to be flase)